src/Jeffijoe.MessageFormat/Jeffijoe.MessageFormat.csproj

```diff
@@ -23,4 +23,17 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
   </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="../Jeffijoe.MessageFormat.MetadataGenerator/Jeffijoe.MessageFormat.MetadataGenerator.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <PluralLanguagesMetadataExcludeLocales>si da is mk ceb fil tl lv prg bs hr sh sr fr dsb hsb lt</PluralLanguagesMetadataExcludeLocales>
```

Comment:
These languages contain constructs that this generator cannot parse yet

Reply:
`da` is Danish? Damn, sucks to be me 😂

Reply:
Eventually, support for these should be there, but I figured that it would be even harder to review with more changes 😅